File: src/main/java/com/telenordigital/prime/firebase/AbstractValueEventListener.java
package com.telenordigital.prime.firebase;

import com.google.firebase.database.DataSnapshot;
import com.google.firebase.database.DatabaseError;
import com.google.firebase.database.ValueEventListener;

public  abstract class AbstractValueEventListener implements ValueEventListener {
    @Override
    public void onDataChange(final DataSnapshot snapshot) {
        // Intentionally left blank.
    }

    @Override
    public void onCancelled(final DatabaseError error) {
        // Intentionally left blank.
    }
}

File: firebase-store/src/main/kotlin/org/ostelco/prime/storage/firebase/AbstractValueEventListener.kt
package org.ostelco.prime.storage.firebase

import com.google.firebase.database.DataSnapshot
import com.google.firebase.database.DatabaseError
import com.google.firebase.database.ValueEventListener

abstract class AbstractValueEventListener : ValueEventListener {
    override fun onDataChange(snapshot: DataSnapshot) {
        // Intentionally left blank.
    }

    override fun onCancelled(error: DatabaseError) {
        // Intentionally left blank.
    }
}
